fe/fe-core/src/main/java/com/starrocks/server/MetadataMgr.java

```diff
@@ -0,0 +1,32 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.server;
+
+import com.starrocks.connector.ConnectorMetadata;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.Optional;
+import java.util.concurrent.ConcurrentHashMap;
+import javax.annotation.concurrent.GuardedBy;
+
+public class MetadataMgr {
```

Comment:
Have we agreed to use `Mgr`? How about full name `Manager`?

Reply:
this is just to keep the style consistent